recipe/meta.yaml

```diff
@@ -29,10 +29,15 @@ requirements:
     - pyzmq
     - regex
     - setproctitle
+    - radical.gtod
```

Comment:
hi @jan-janssen, is there a reason you want to add `radical.gtod` here? It is vice versa - `radical.gtod` depends on `radical.utils`

Reply:
Somehow I thought it was in the requirements, but you are right it is not.